index.md

```diff
@@ -2,9 +2,343 @@
+The solution was to modify the Butler software to have a web service that runs inside the Rubin Science Platform {cite:p}`LDM-542` and which can look at the authorization tokens {cite:p}`DMTN-182` and decide what the user can see in the Butler repository wherever the database and data reside.
+
+A client/server butler was prototyped in {cite:p}`DMTN-242` and a design meeting was held in Princeton in 2023 October {cite:p}`DMTN-282` to decide on an initial implementation plan.
+Since that meeting, work on client/server has progressed to the extent that a core system can be deployed on the RSP and there is support for retrieving datasets using signed URLs.
+A meeting was held at SLAC at the Joint Technical Meeting in early 2024 February with representatives from the Data Facilities and architecture teams to discuss different options for deployment and implementation strategies for client/server at the various sites.
+This document summarizes the outcomes of that meeting.
+
+In this document the following terms are used:
+
+* "direct butler" is a butler instance that communicates directly with a database and a file store.
+* "remote butler" is a butler instance that communicates with a web server process and might require a proxy or signed URL to retrieve file contents.
+
+## Core Assumptions
+
+There are some core assumptions made by the developer team that should be spelled out explicitly to ensure that there is a shared understanding.
+
+* A `Butler` can be instantiated from a URI to a configuration file without the user having to know whether they are connecting directly to a butler registry or via a client/server.
+* Data Release Processing, including multi-site processing {cite:p}`DMTN-213`, will use a direct butler.
+  Campaign processing is run by trusted users doing large graph builds.
+* Any transfers of Prompt Processing products to butler repositories at the USDF will use direct butler.
+* Developers logged in directly to the USDF will use direct butler.
+  Permissions and auditing will be handled by creating user accounts in the PostgreSQL databases rather than the current use of shared accounts.
+* Summit users do not need to access summit butlers using client/server and their notebooks can connect directly to Butler.
+* All RSP deployments will require a client/server butler to implement VO services, but that server can be restricted to only those services.
+* External Data Rights holders must always access butler datasets through client/server.
+* It will be possible for a server to sign URLs such that datasets can be retrieved directly by clients without requiring a proxy server.
+  (see also {cite:p}`DMTN-284`)
+
+### Data Preview 0
+
+When Data Preview 0.2 {cite:p}`RTN-041` is relocated from the Interim Data Facility to the USDF in August it will no longer be generally accessible to the data preview delegates.
```

Comment:
What I really mean is that we are not going to use the August timeline as a milestone for deployment of a fully functional client/server butler at USDF so that people without SLAC accounts can still access DP0.2. At such a time as we do have a functional read-only client/server butler that can work in hybrid mode from data.lsst.cloud and that can sign URLs from SLAC then we could consider giving delegates access to it again.

Reply:
I wrote down what @drphilmarshall told me at the meeting. It's clear now that this is not what is going to happen and so that entire section has to be removed because it's no longer saying anything.

Reply:
I've removed the timeline and simply stated that DP0.2 availability is not driving client/server timelines.